Components/ListSearchResults/Partials/Results/_style.scss

```diff
@@ -49,8 +55,16 @@
   }
 
   .excerpt {
-    color: #9b9b9b;
+    font-size: 0.875rem;
     margin-bottom: 0;
     padding-bottom: 0;
   }
 }
+
+.postTitle {
+  font-size: 1rem;
+}
+
+.postLink {
```

Comment:
These 2 classes don't look like naming conventions according to maintainable css to me. Please refer to https://maintainablecss.com/chapters/modules/.